### PR TITLE
Fix constant expressions.

### PR DESCRIPTION
**Pull request by ptersilie**
See first commit message for details.

### CODE REVIEW

**Comment by vext01** [comment]
>  it does not remap the global variable, since we haven't copied it into JITMod yet.

If you copy the global first, can we avoid the dance then?

File: c_tests/tests/constexpr.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Compiler:
+// Run-time:
+
+#include <assert.h>
```
Comment (vext01) [inline c_tests/tests/constexpr.c:4]: Can we add a comment saying what the test checks?

File: c_tests/tests/constexpr.c [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Compiler:
+// Run-time:
```
Comment (vext01) [inline c_tests/tests/constexpr.c:2]: Did we want to check for the presence of a constant expr somewhere in the module?

Comment (ptersilie) [reply c_tests/tests/constexpr.c:2]: Fixed in 5ec3b32.

File: c_tests/tests/constexpr.c (hunk 1 above)
Comment (ptersilie) [reply c_tests/tests/constexpr.c:16]: Fixed in e4ccdcf.

**Comment by vext01** [comment]
LGTM. Please squash.

**Comment by ptersilie** [comment]
Squashed.

**Comment by vext01** [comment]
Needs cfmt.

**Comment by ptersilie** [comment]
Fixed.

**Pull request merged**